### DISCUSSION TITLE
码表与词典

### DISCUSSION THREAD

**Discussion by tristone13th** [rime/home#1052] [Q&A] [answered]
Hello，新人还是没有看明白码表与字典的定义是什么，感觉都是码空间到值空间的映射，请问它们之间有什么区别吗？

**Comment by LEOYoon-Tsaw** [comment, marked as answer]
同義

**Reply by lotem** [reply]
同意 ：）

**Comment by lotem** [comment]
如果要細分，可以講「（用戶編寫的）碼表 + 詞典配置 +（導入詞彙｜自動推導編碼）= 詞典」。

**Reply by tristone13th** [reply]
> 如果要細分，可以講「（用戶編寫的）碼表 + 詞典配置 +（導入詞彙｜自動推導編碼）= 詞典」。

用户编写的码表是否等于用户编写的 dict.yaml 文件里定义的编码 + dict.yaml 文件里导入的编码；而导入词条是否等于 custom_phrase.txt 中所定义的短语？
